<ui version="4.0" >
 <author>Diego Iastrubni &lt;diegoiast@gmail.com></author>
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Editor options</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="autoBrackets" >
           <property name="text" >
            <string>Auto brakets</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QCheckBox" name="matchBrackets" >
           <property name="text" >
            <string>Match brackets</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="showLineNumbers" >
           <property name="text" >
            <string>Show line numbers</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QCheckBox" name="showWhiteSpaces" >
           <property name="text" >
            <string>Show white spaces</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="useSmartHome" >
           <property name="text" >
            <string>Use smart home</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QCheckBox" name="markCurrentLine" >
           <property name="text" >
            <string>Mark current line</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QCheckBox" name="wrapLines" >
           <property name="text" >
            <string>Wrap long lines</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>412</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="showMargins" >
           <property name="text" >
            <string>Show margins</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="marginSize" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="maximum" >
            <number>1024</number>
           </property>
           <property name="value" >
            <number>80</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Tab size</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="tabSize" >
           <property name="value" >
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Brackets list</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="matchCraketsList" >
           <property name="text" >
            <string>(){}[]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="labelFontPreview" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>0</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
             <property name="midLineWidth" >
              <number>1</number>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
             <property name="margin" >
              <number>2</number>
             </property>
             <property name="indent" >
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnChooseFont" >
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Available color schemes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorsCombo" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QsvEditor" name="sampleEdit" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsvEditor</class>
   <extends>QTextEdit</extends>
   <header>qsveditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>autoBrackets</tabstop>
  <tabstop>markCurrentLine</tabstop>
  <tabstop>showLineNumbers</tabstop>
  <tabstop>showWhiteSpaces</tabstop>
  <tabstop>matchBrackets</tabstop>
  <tabstop>matchCraketsList</tabstop>
  <tabstop>tabSize</tabstop>
  <tabstop>btnChooseFont</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>colorsCombo</tabstop>
  <tabstop>sampleEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showMargins</sender>
   <signal>clicked(bool)</signal>
   <receiver>marginSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>85</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
